### DISCUSSION TITLE
Change OLED brightness

### DISCUSSION THREAD

**Discussion by pirx242** [supersaiyanmode/PyWebOSTV#79] [General]
Hey!
This is great, i'll use this package for a few things now! :)
Just one question:)
How could i find out if its at all possible to change the oled brightness of the TV? I mean, with the ssap protocol i guess.
There are no comprehensive docs of what ssap requests webos accepts, are there possibly? :)
  wbr / Alex

**Comment by supersaiyanmode** [comment]
Hi, to answer both your questions: There isn't any comprehensive doc around SSAP APIs. I wrote the initial version by looking at ConnectSDK (whick apparently was written by LG themselves) because I felt there is a better way to scale new APIs as they become known. We look at other libraries to see if they have somehow reverse-engineered or found new APIs. If they did -- we try to bring them here.

So essentially, if you see OLED brightness APIs being exposed by other libraries, feel free to submit a PR. I'll try to upload to PyPI ASAP.

**Reply by supersaiyanmode** [reply]
@pirx242 Quick update: Please see: https://www.webosose.org/docs/reference/ls2-api/com-webos-service-settings/#example-14.

Can you please try playing around it? I've seen this pattern where you can replace luna:// with ssap://, and with an identical payload -- it could likely work! Can you please try this out and tell us if it does end up working?

**Reply by pirx242** [reply]
Ah, great! I hope to have a look at that this weekend! :)
